### DISCUSSION TITLE
[Question] How can I do sign-in from the frontend when using the Express integration?

### DISCUSSION THREAD

**Discussion by vedantroy** [nextauthjs/next-auth#12317] [Help]
I have a React app that is built with Vite. I also have an express server that is using the Auth.js integration. How can I sign-in from the React app (Google OAuth), using the express server?

Similar to: https://github.com/nextauthjs/next-auth/discussions/12306.

**Comment by Randulfe** [comment]
Would also be interested in some documentation for these kind of scenarios as you can end up hitting many roadblocks with callback urls and cookie/cors policies.

**Comment by Randulfe** [comment]
Maybe [this](https://github.com/nextauthjs/next-auth/discussions/12306#discussioncomment-11421092) can help you @vedantroy

**Comment by guptaashwanee** [comment]
> I have a React app that is built with Vite. I also have an express server that is using the Auth.js integration. How can I sign-in from the React app (Google OAuth), using the express server?
> 
> Similar to: #12306.

I am trying to implement the same setup for my college project, did you got any success in this
